Datasnap/Classes/Event Properties/Identifier.m

```diff
@@ -20,7 +20,7 @@ @implementation Identifier
 @synthesize webUserFingerprint;
```

Comment:
@almcintyre : why are we synthesizing these variables if we are always using 'self' to reference them? The synthesize call is unneeded in these cases.